### ISSUE TITLE
Let´s integrate country specific search engines

### ISSUE THREAD

**Issue by Turricaner** [universeroc/super-drag#21]
Gratulation for this MEGA Extension.

But can you enhance this extension with the function to
integrate country specific search engines ?

In Germany:
- Amazon.de
- https://de.wikipedia.org

and so on ...

**Comment by universeroc** [comment]
> Gratulation for this MEGA Extension.
> 
> But can you enhance this extension with the function to integrate country specific search engines ?
> 
> In Germany:
> 
> * Amazon.de
> * https://de.wikipedia.org
> 
> and so on ...

Yes! sorry for late reply. The open search engine configuration can solve this problem. I'll open it soon.

**Comment by universeroc** [comment]
you can make it by custom engine in v1.0
